rero_ils/modules/utils.py

```diff
@@ -990,3 +990,45 @@ def requests_retry_session(retries=3, backoff_factor=0.3,
     session.mount('http://', adapter)
     session.mount('https://', adapter)
     return session
+
+
+class JsonWriter(object):
+    """Json Writer."""
+
+    count = 0
+
+    def __init__(self, filename, indent=2):
+        """Constructor.
+
+        :params filename: file name of write file.
```

Comment:
File name of the file to be written? File name of the written file? Something else?